### PR TITLE
Issue/2298 add cache at policy level

### PR DESCRIPTION
**Pull request by gcusnieux**
fix gravitee-io/issues#2298

### CODE REVIEW

File: stories/atoms/gv-autocomplete.stories.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const conf = {
 export const BasicUsage = makeStory(conf, {
   items: [
     {
-      innerHTML: '<gv-input placeholder="Type something..."></gv-input>',
+      innerHTML: '<gv-input placeholder="Type something..." description="Try to ype something"></gv-input>',
```
Comment (phiz71) [inline stories/atoms/gv-autocomplete.stories.js:51]:
```suggestion
      innerHTML: '<gv-input placeholder="Type something..." description="Try to type something"></gv-input>',
```

**Review by phiz71 — CHANGES_REQUESTED** [review]

**Review by phiz71 — APPROVED** [review]

File: src/atoms/gv-input.js [hunk 2]
```diff
@@ -42,6 +42,7 @@ import { shapeClipboard, shapeCopied } from '../styles/shapes';
  * @attr {String} title - title of the input
  * @attr {String} name - name of the input
  * @attr {String} placeholder - an example value to display in the input when empty
+ * @attr {String} description - an description
```
Comment (ytvnr) [inline src/atoms/gv-input.js:45]:
```suggestion
 * @attr {String} description - a description
```

File: src/atoms/gv-select.js [hunk 3]
```diff
@@ -44,6 +44,7 @@ import { withResizeObserver } from '../mixins/with-resize-observer';
  * @attr {String} title - title of the select
  * @attr {String} name - name of the select
  * @attr {String} placeholder - an example value to display in the select when empty
+ * @attr {String} description - an description
```
Comment (ytvnr) [inline src/atoms/gv-select.js:47]:
```suggestion
 * @attr {String} description - a description
```

File: src/molecules/gv-expression-language.js [hunk 4]
```diff
@@ -31,6 +31,7 @@ import { dispatchCustomEvent } from '../lib/events';
  * @attr {String} value - the value of the input
  * @attr {String} label - label of the input
  * @attr {String} placeholder - an example value to display in the input when empty
+ * @attr {String} description - an description
```
Comment (ytvnr) [inline src/molecules/gv-expression-language.js:34]:
```suggestion
 * @attr {String} description - a description
```

**Review by ytvnr — CHANGES_REQUESTED** [review]
some typos

File: src/atoms/gv-autocomplete.js [hunk 5]
```diff
@@ -355,13 +364,23 @@ export class GvAutocomplete extends LitElement {
...
+  _close() {
+    this._forceOpen = false;
+  }
+
+  _onFocus() {
+    this._open();
   }
 
   _onBlur() {
+    // Important when use custom HTML in options
```
Comment (ytvnr) [inline src/atoms/gv-autocomplete.js:381]:
```suggestion
    // Important when using custom HTML in options
```

**Review by ytvnr — APPROVED** [review]
just a typo in a comment

**Pull request merged**